invenio_rdm_records/services/communities/service.py

```diff
@@ -88,18 +94,26 @@ def _include(self, identity, community_id, comment, require_review, record, uow)
             {"community": community_id}
         ).resolve()
 
-        data = {"payload": {"content": comment, "format": "html"}} if comment else {}
+        comment_schema = self._wrap_schema(CommentEventType.marshmallow_schema())
+        comment_data = {}
+
+        if comment:
+            comment_data, errors = comment_schema.load(
+                comment,
+                context={"identity": identity},
+            )
+
         request_item = current_requests_service.create(
             identity,
-            data,
+            comment_data,
             type_,
             receiver,
             topic=record,
             uow=uow,
         )
         # create review request
         request_item = current_rdm_records.community_inclusion_service.submit(
-            identity, record, community, request_item._request, data, uow
+            identity, record, community, request_item._request, comment_data, uow
```

Comment:
I think that the comment should not be added 2 times, but probably only here (and not in the `create`). To be checked.

Reply:
Right, however we have to pass the data anyway, I changed it and passed an empty object on create.